Minor container publish issue with 4.0.0-beta.5

```bash
My.Database.Internal net9.0 failed with 1 error(s) (4,9s) → bin\Release\net9.0\My.Database.Internal.dacpac
    C:\Program Files\dotnet\sdk\10.0.102\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(371,5): error MSB3030: Could not copy the file "bin\Release\net9.0\My.Database.Internal.xml" because it was not found.
```

It was because I had this in my Directory.Build.Props file:

`<GenerateDocumentationFile>true</GenerateDocumentationFile>`

Adding this to my database project fixed the issue:

```xml
<Project Sdk="MSBuild.Sdk.SqlProj/4.0.0-beta.5">
    <PropertyGroup>
      <GenerateDocumentationFile>false</GenerateDocumentationFile>

```

--- a/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props
+++ b/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props
@@ -12,6 +12,9 @@
     <TargetExt>.dacpac</TargetExt>
     <EnableSdkContainerSupport Condition="'$(EnableSdkContainerSupport)' == ''">true</EnableSdkContainerSupport>
     <SqlPackageDownloadUrl>https://aka.ms/sqlpackage-linux</SqlPackageDownloadUrl>
+
+    <!-- Setting this here so that we won't try to generate a documentation file and then fail on publish since it doesn't exist -->
+    <GenerateDocumentationFile>False</GenerateDocumentationFile>
   </PropertyGroup>
 
   <PropertyGroup>

